doc/how_to_guides/chomp_planner/launch/chomp_demo.launch.py

```diff
@@ -0,0 +1,158 @@
+import os
+import yaml
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument
+from launch.substitutions import LaunchConfiguration
+from launch.conditions import IfCondition, UnlessCondition
+from launch_ros.actions import Node
+from launch.actions import ExecuteProcess
+from ament_index_python.packages import get_package_share_directory
+from moveit_configs_utils import MoveItConfigsBuilder
+
+
+def generate_launch_description():
+
+    # Command-line arguments
+    tutorial_arg = DeclareLaunchArgument(
+        "rviz_tutorial", default_value="False", description="Tutorial flag"
+    )
+
+    db_arg = DeclareLaunchArgument(
+        "db", default_value="False", description="Database flag"
+    )
+
+    moveit_config = (
+        MoveItConfigsBuilder("moveit_resources_panda")
+        .robot_description(file_path="config/panda.urdf.xacro")
+        .robot_description_semantic(file_path="config/panda.srdf")
+        .trajectory_execution(file_path="config/gripper_moveit_controllers.yaml")
+        .planning_pipelines(pipelines=["ompl", "chomp"])
+        .to_moveit_configs()
+    )
+
+    # Start the actual move_group node/action server
+    move_group_node = Node(
+        package="moveit_ros_move_group",
+        executable="move_group",
+        output="screen",
+        parameters=[moveit_config.to_dict()],
+        arguments=["--ros-args", "--log-level", "info"],
+    )
+
+    # RViz
+    tutorial_mode = LaunchConfiguration("rviz_tutorial")
+    rviz_base = os.path.join(get_package_share_directory("moveit2_tutorials"), "config")
+    rviz_full_config = os.path.join(rviz_base, "moveit_chomp.rviz")
+    rviz_empty_config = os.path.join(rviz_base, "moveit_chomp.rviz")
```

Comment:
If these lines are meant to be the same, do we need to have two configuration variables and the IfCondition for `rviz_node_tutorial` and `rviz_node`?